histomicstk/segmentation/label/trace_object_boundaries.py

```diff
@@ -6,7 +6,8 @@
 
 def trace_object_boundaries(im_label,
                             conn=4, trace_all=False,
-                            x_start=None, y_start=None, max_length=None):
+                            x_start=None, y_start=None,
+                            max_length=None, eps_colinear_area=1.0):
```

Comment:
If we set the default colinear area to a small value, then this works fine.  I recommend something like `0.01` or `0.1`.